tests/utils/utils.py

```diff
@@ -165,3 +167,26 @@ def magic(path, *args, **kwargs):
     env["KUBERNETES_PORT_443_TCP_PORT"] = "123"
     env["HOSTNAME"] = "test"
     return env
+
+
+def retry(retries: int = 3, delay: int = 1) -> Callable:
```

Comment:
is this function used anywhere?

Reply:
No (it was used in something that turned out to be unnecessary). do you think I should remove it?

Reply:
hmmm... whatever you want... :)

Reply:
I thought it might be useful one day in some tests, especially where we're relying on naping

Reply:
we have the retry instruments in `lib`, but they are too heavy/complex for cases like that

Reply:
actually, let me yank it since it makes codecov go down :)